core/src/main/java/org/mule/runtime/core/internal/util/rx/Operators.java

```diff
@@ -78,6 +89,42 @@ public void onComplete() {
     }
   }
 
+  /**
+   * Register reactor hooks using the provided class. This needs to use reflection because reactor could be
+   * loaded on different class loaders.
+   *
+   * In reactor 3.1, the error handler will be explicitly defined for each flux instead of globally.
+   */
+  public static void setupErrorHooks(Class clazz) {
```

Comment:
Also, if it's not in API then it won't be available for other plugins/services to load it.